### PR TITLE
New package: apache-maven-bin-3.3.3

### PR DESCRIPTION
**Pull request by Spencer-H**

### CODE REVIEW

File: srcpkgs/apache-maven/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'apache-maven'
+pkgname=apache-maven
+version=3.3.3
+revision=1
+depends="openjdk"
```
Comment (xtraeme) [inline srcpkgs/apache-maven/template]: This is wrong... should be `depends="java-environment"` if this needs the JDK, or `depends="java-runtime"` if it needs JRE.

**Comment by Spencer-H** [comment]
When I try to package it with either of those selected it simply spits out: 
`   [runtime] java-runtime>=0: not found. ` and then
`=> ERROR: xbps-src: unexistent file: /void-packages/srcpkgs/java-runtime/template`
I'm probably just missing something, but I'm not sure what.

**Comment by thypon** [comment]
This is the apache-maven bin version. Should be named apache-maven-bin instead?

**Comment by Spencer-H** [comment]
Sure that makes sense. Though It's beyond my ability to package the source one, so it would be the only one in here for now.

**Comment by pullmoll** [comment]
`vmkdir usr/bin` without the leading slash is the correct syntax.

**Comment by Spencer-H** [comment]
The syntax for vmkdir should now be correct.

**Pull request merged**